# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rt61/no-depmod.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur rt61-1.1.0-b1/Module/Makefile rt61-1.1.0-b1-fixed/Module/Makefile
--- rt61-1.1.0-b1/Module/Makefile	2006-06-18 10:58:39.000000000 +0200
+++ rt61-1.1.0-b1-fixed/Module/Makefile	2006-10-01 13:56:24.000000000 +0200
@@ -121,11 +121,9 @@
 	@echo "install '$(MODULE_OBJECT)' to $(MODULE_ROOT)"
 	install -m 755 -o 0 -g 0 -d $(MODULE_ROOT)
 	install -m 644 -o 0 -g 0 $(MODULE_OBJECT) $(MODULE_ROOT)
-	/sbin/depmod -a
 else
 	echo "2.6 module install"
 	make $(KBUILD_PARAMS) modules_install
-	/sbin/depmod -a
 endif
 
 install: modules_install
